### PR TITLE
add support for resource and operation defaults

### PR DESCRIPTION
**Pull request by tomjelinek**

### CODE REVIEW

**Comment by tomjelinek** [comment]
[citest]

File: tests/tests_cib_rsc_op_defaults.yml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+              register: __test_pcs_rsc_defaults_config
+              changed_when: false
+
+            - name: Print real resource defaults configuration
+              debug:
+                var: __test_pcs_rsc_defaults_config
+
+            - name: Print expected resource defaults configuration
+              debug:
+                var: __test_expected_lines | select | list
```
Comment (richm) [inline tests/tests_cib_rsc_op_defaults.yml:115]: do you need the `select` here?  afaict, all of the expected lines have a non-null and non-empty value

Comment (tomjelinek) [reply tests/tests_cib_rsc_op_defaults.yml]: You are right, it's not needed. I removed it.

**Comment by tomjelinek** [comment]
[citest]

**Review by richm — APPROVED** [review]

**Comment by richm** [comment]
Is this failure ok?  https://dl.fedoraproject.org/pub/alt/linuxsystemroles/logs/lsr-citool_ha_cluster-114-46325a5_Fedora-36_20230327-121129/artifacts/tests_qdevice_minimal-FAILED.log
```
TASK [Get quorum device status] ************************************************
task path: /WORKDIR/dist-git-ha_cluster-rsc-op-defaults-o6ii7m08/tests/tasks/assert_qdevice_in_cluster.yml:14
Monday 27 March 2023  11:19:33 +0000 (0:00:00.081)       0:03:10.395 ********** 
ok: [sut] => {
    "changed": false,
    "cmd": [
        "pcs",
        "--",
        "quorum",
        "device",
        "status"
    ],
    "delta": "0:00:00.596996",
    "end": "2023-03-27 11:19:34.232150",
    "rc": 0,
    "start": "2023-03-27 11:19:33.635154"
}

STDOUT:

Qdevice information
-------------------
Model:			Net
Node ID:		1
Configured node list:
    0	Node ID = 1
Membership node list:	1

Qdevice-net information
----------------------
Cluster name:		test-cluster
QNetd host:		localhost:5403
Algorithm:		LMS
Tie-breaker:		Node with lowest node ID
State:			Connect failed

TASK [Assert qdevice connected] ************************************************
task path: /WORKDIR/dist-git-ha_cluster-rsc-op-defaults-o6ii7m08/tests/tasks/assert_qdevice_in_cluster.yml:21
Monday 27 March 2023  11:19:34 +0000 (0:00:00.911)       0:03:11.307 ********** 
fatal: [sut]: FAILED! => {
    "assertion": "'State:\t\t\tConnected' in __test_quorum_device_status.stdout",
    "changed": false,
    "evaluated_to": false
}

MSG:

Assertion failed
```
some sort of test flake?

**Comment by tomjelinek** [comment]
> Is this failure ok? https://dl.fedoraproject.org/pub/alt/linuxsystemroles/logs/lsr-citool_ha_cluster-114-46325a5_Fedora-36_20230327-121129/artifacts/tests_qdevice_minimal-FAILED.log

I'm not sure what's going on there. I am unable to reproduce it on my machines. And it passed on Fedora 36 with ansible-2.14, even though ansible version shouldn't make a difference. The changes in this PR are unrelated to that test.

**Pull request merged**